.editorconfig

```diff
@@ -21,7 +21,7 @@ ij_kotlin_code_style_defaults = KOTLIN_OFFICIAL
 ij_kotlin_name_count_to_use_star_import = 99
```

Comment:
NIT: is this file supposed to be in the repo?

Reply:
Yes